tests/cases/image_geometry.60

```diff
@@ -45,14 +45,15 @@ TestCase := Rectangle {
     property <bool> image_in_layout_custom_fit_ok: image_in_layout_with_explicit_fit.image-fit == ImageFit.fill;
     property <bool> image_with_missing_height_ok <=> image_with_missing_height.expected_height_ok;
     property <bool> image_with_missing_width_ok <=> image_with_missing_width.expected_width_ok;
+
+    property <bool> test: fixed_image_default_image_fit_ok && fixed_image_image_fit_override_ok && image_in_layout_fit_ok && image_in_layout_custom_fit_ok && image_with_missing_height_ok && image_with_missing_width_ok;
```

Comment:
There's something for future assert syntax perhaps, that we could express here, use from the interpreter (as opposed to querying for the all-combining test property) as well as from the generated code.

Reply:
yeah, but that's not yet implemented.
But when this fails, the test driver will print out all the property to the debug output so we'll know what the problem is.